functions/scaler/BellhopScaler/run.ps1

```diff
@@ -3,21 +3,22 @@ param($QueueItem, $TriggerMetadata)
 
 function Initialize-TagData {
     param (
-        $inTags
+        $inTags,
+        $tagMap
```

Comment:
Could you maybe add a comment explaining what $inTags and $tagMap represent? Just a small comment explaining what each value is.

Reply:
This will likely come later, but I do completely agree the Scaler code needs to be documented/commented in much greater detail.